internal/jujuclient/modelmanager.go

```diff
@@ -325,3 +327,14 @@ func (c Connection) ChangeModelCredential(ctx context.Context, model names.Model
 	}
 	return out.OneError()
 }
+
+// ListModels returns the models that the specified user
+// has access to in the current server.  Only that controller owner
+// can list models for any user (at this stage).  Other users
+// can only ask about their own models.
+//
+// In our wrapper, we ask for the owner. So expect ALL models from
+// the controller.
+func (c Connection) ListModels(ctx context.Context) ([]base.UserModel, error) {
+	return modelmanager.NewClient(&c).ListModels("admin")
```

Comment:
learning question:
if a local user creates a model, and then the controller is migrated to jaas.
What would be the owneridentity of that model under the juju controller? And under JIMM?

Reply:
We have a flag on import model to change the owner from a local user to an external user. This call specifically though is getting all models, so we use our auth system to filter and not juju.

Reply:
jimm does not care about models created by local users.. unless we intentionally import the model.

Reply:
ty